src/cray/boa/preflight.py

```diff
@@ -133,7 +133,7 @@ def check_cfs(self):
         return self.check_uri(CFS_ENDPOINT)
 
     def check_capmc(self):
-        return self.check_uri(os.path.join(CAPMC_ENDPOINT, 'get_node_rules'),)
+        return self.check_uri(os.path.join(CAPMC_ENDPOINT, 'health'),)
```

Comment:
Did they deprecate get_node_rules, too? 
Regardless, 'health' seems like a better pre-flight endpoint to check than get_node_rules.

Reply:
https://github.com/Cray-HPE/docs-csm/blob/main/introduction/CAPMC_deprecation.md